warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -2098,3 +2098,20 @@ models:
       - *rt_trip_max_datetime_pacific
       - *rt_trip_start_time_interval
       - *rt_trip_num_distinct_updates
+  - name: fct_monthly_shapes
+    description: |
+      An aggregation of GTFS schedule shapes by month, filtered to the shape_id associated with each route_id with the most trips.
+    columns:
+      - name: source_record_id
+      - name: name
+      - name: base64_url
+      - name: route_id
+      - name: shape_id
+      - name: month
+        description: |
+          Actual calendar month (Pacific Time dates) in which this service was scheduled to occur.
```

Comment:
If you are using `service_date`, then this is not necessarily an actual Pacific Time calendar month, this is the dates within that month as defined by the agency's definition of a service date. I still think `service_date` is probably the best way to do this, but would just update this to say `service month`